go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/lanrat/czds
 
 go 1.13
+
```

Comment:
remove this since its not needed anymore.

Reply:
done